docs/index.md

```diff
@@ -30,36 +30,25 @@ For an overview of all available inputs, see [this page](./inputs.html).
   "GatkVariantCalling.referenceFastaDict": "The dict file for the reference fasta",
   "GatkVariantCalling.dbsnpVCF": "A dbSNP VCF file",
   "GatkVariantCalling.dbsnpVCFIndex": "The index (.tbi) for the dbSNP VCF file",
-  "GatkVariantCalling.bamFiles": "A list of input BAM files and their associated indexes"
-}
+  "GatkVariantCalling.bamFilesAndGenders": "A list tuples of bam files, their indexes and the gender of the sample. The gender is optional. " 
+ }
 ```
+`gender` can be set to `null`. Actionable values are `female`, `male`, `F`, 
+`f`, `M`, or `m`. Any other values and `null` will handle the sample as an 
+"unknown" gender (both X and Y are called).
```

Comment:
Will both X and Y be called as diploid when the gender is set to Unknown?

Reply:
Clarified this in the documentation.